src/popup.html

```diff
@@ -30,38 +30,63 @@
         .shadow {
             color: dimgray;
             display: block;
+            margin-bottom: 5px;
         }
+
+       select {
+           margin-top: 5px;
+           font-size: inherit;
+           border-radius: 5px;
+       }
+
+       .percentagesPosition {
+           margin-left: 23px;
+           margin-bottom: 10px;
+       }
     </style>
 </head>
 
 <body style="width: 250px;">
-<div style="margin-bottom: 10px;">
-    <img class="center" src="../img/icon32.png">
-    <h3 class="title">BetterMaplewood</h3>
-</div>
-<hr>
-<div>
-    <input checked data-size="mini" data-toggle="toggle" id="quickview" type="checkbox">
-    <label for="quickview">Enable mark quickview</label>
-</div>
-<div>
-    <input checked data-size="mini" data-toggle="toggle" id="calculation" type="checkbox">
-    <label for="calculation">Enable average calculation</label>
-</div>
-<div>
-    <input checked data-size="mini" data-toggle="toggle" id="liveModification" type="checkbox">
-    <label for="liveModification">Enable live mark modification</label>
-</div>
-<div>
-    <input checked data-size="mini" data-toggle="toggle" id="percentageColumn" type="checkbox">
-    <label for="percentageColumn">Enable percentage column</label>
-</div>
-<hr>
-<div class="footer">
-    Created with ❤️ by Youssef Soliman
-    <span class="shadow" id="version"></span>
-</div>
-<script src="popup.js"></script>
+    <div style="margin-bottom: 10px;">
+        <img class="center" src="../img/icon32.png">
+        <h3 class="title">BetterMaplewood</h3>
+    </div>
+    <hr>
+    <div>
+        <input checked data-size="mini" data-toggle="toggle" id="quickview" type="checkbox">
+        <label for="quickview">Enable mark quickview</label>
+    </div>
+    <div>
+        <input checked data-size="mini" data-toggle="toggle" id="calculation" type="checkbox">
+        <label for="calculation">Enable average calculation</label>
+    </div>
+    <div>
+        <input checked data-size="mini" data-toggle="toggle" id="liveModification" type="checkbox">
+        <label for="liveModification">Enable live mark modification</label>
+    </div>
+    <div>
+        <input checked data-size="mini" data-toggle="toggle" id="betterTableLayout" type="checkbox">
+        <label for="betterTableLayout">Enable better table layout</label>
+    </div>
+    <div>
+        <input checked data-size="mini" data-toggle="toggle" id="percentages" type="checkbox">
+        <label for="percentages">Enable percentage column</label>
+        <div class="percentagesPosition">
+            <label for="percentagesPosition">Position: </label>
+            <select id="percentagesPosition">
+                <option>Last column</option>
+                <option>Second column</option>
```

Comment:
It was supposed to be third column instead of last column, since there is so much confusion around this, I recommend having a numbered box instead (1-6) (this would make your logic much easier and dynamic in the code, the selector can be based directly off this number) or just adding third column as an option.